docs/python/onboarding.md

```diff
@@ -0,0 +1,187 @@
+
+- Customize your app's look and feel
+- Adding a background-image
+- Customize the admin form
+- Adding related objects
+- Customize the admin change list
+- Customize the admin look and feel
+- Customize the admin index page
+
+---
+
+## Day 5 - Django Rest Framework Part 1, 2 and 3
+
+[Django REST Framework Tutorial](https://www.django-rest-framework.org/tutorial/1-serialization/) (Part 1, 2)
+[Part 1](https://www.django-rest-framework.org/tutorial/1-serialization/)
+
+- Setting up a new environment
+- Working with Models
+- Working with Serializers
+- Writing regular Django views using our Serializer
+- Testing our first attempt at a Web API
+
+[Part 2](https://www.django-rest-framework.org/tutorial/2-requests-and-responses/)
+
+- Requests and Responses
+
+[Part 3](https://www.django-rest-framework.org/tutorial/3-class-based-views/)
+
+- Class Based Views
+- Mixins
+- Generic Class Based Views
+
+---
+
+## Day 6 - Django Rest Framework Part 4, 5 and 6
+
+[Django REST Framework Tutorial](https://www.django-rest-framework.org/tutorial/4-authentication-and-permissions/)
+(Part 4, 5)
+[Part 4](https://www.django-rest-framework.org/tutorial/4-authentication-and-permissions/)
+
+- Adding permissions to views
+- Create a Login API
+- Object level permissions
+- Authenticating with the API
+
+[Part 5](https://www.django-rest-framework.org/tutorial/5-relationships-and-hyperlinked-apis/)
+
+- Adding a root API
+- Hyperlinking our API
+- Adding pagination
+
+[Part 6](https://www.django-rest-framework.org/tutorial/6-viewsets-and-routers/)
+
+- Using ViewSets
+- Using routers
+- Adding pagination
+
+---
+
+## Day 7 - Pandas
+
+[pandas getting started](https://pandas.pydata.org/docs/getting_started/index.html)
+
+- Installing pandas
+- pandas overview
+- Getting started tutorials
+
+---
+
+## Day 8 - Project
+
+This project consists on the development of a Rest api to manage/views: Authors, Books and Categories.
+
+### Requirements
+
+- User needs to be able to manage Authors, Books, and Categories in the app.
+- Each Author can have many Books that he/her has written and each book can be included in multiple categories.
+- The User should be able to view lists of Authors and Books.
+- The Books should be able to be filtered by Author and byCategory
+
+> Optional: The App should also include a page to view some basic statistics, like the number of Books per Author, or the number of Books per Category.
+
+Optional: To complicate the models. A book can have many instances and users can request an instance to take home with a
+requested date
+
+### Acceptance Criteria
+
+- Design the model entity relation for this project
+    - use [Mermaid](https://mermaid.js.org/syntax/entityRelationshipDiagram.html), this is supported out of the box by Github's Markdown
+- Design the API endpoints, including:
+    - path
+    - request
+    - response
+
+Once the design/planning part has been taken care and agreed, please create tickets/issues for each of the tasks.
```

Comment:
As a suggestion, wouldn't be interesting to add examples of commits with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) related to the tickets/issues already created:
 - task/<ticket>/task-in-the-branch
 - bugfix/<ticket>/task-in-the-branch
 
Additionally, examples of ticket creation